docs/sphinx/using/simulators.rst

```diff
@@ -90,3 +90,70 @@ options to :code:`nvq++`
 .. code:: bash 
 
     nvq++ --target tensornet src.cpp ...
+
+GPU Simulation Requirements
+==================================
+
+For programmers using the CUDA Quantum Python wheels, extra configuration is needed to install
+our GPU enabled simulation backends.
+
+System Level Dependencies
+++++++++++++++++++++++++++
+
+The system level dependencies for these backends include:
+* An NVIDIA GPU with compute capability 7.0+
+* Driver: Linux (450.80.02+ for CUDA 11, 525.60.13+ for CUDA 12)
+* CUDA Toolkit 11.x or newer
+* Python 3.9+
+
+For more information on the installation of NVIDIA Driver's for Ubuntu, see
+`here <https://help.ubuntu.com/community/NvidiaDriversInstallation>`__.
+
+For installation of the CUDA toolkit, programmers have two options. The first
```

Comment:
I would move this to the install.rst starting from this line (the lines before that are covered by Eric's PR that states the compatibility). It's probably easiest if I create a PR to show what I mean.